include/crpropa/module/SynchrotronRadiation.h

```diff
@@ -53,11 +53,25 @@ class SynchrotronRadiation: public Module {
 	 */
 	SynchrotronRadiation(double Brms = 0, bool havePhotons = false, double thinning = 0, int nSamples = 0, double limit = 0.1);
 	
+	// set the target photon field
 	void setField(ref_ptr<MagneticField> field);
+
+	// set the rms value of the magnetic field (no 3d field is used)
```

Comment:
I would write root-mean square (rms) to make it clear.

Reply:
fixed